virt-* rejects names with space character, even though libvirt accepts it

**Distro**: 
```shell
                 'ooo'                     OS: Artix Linux x86_64 dinit
               'ooxxxoo'                   Kernel: Linux 6.12.0-rc4-1-cachyos-rc
            ':;:iiiioxxxoo'                Shell: fish 3.7.1
          '-.      `':;jiooo'              DE: Mate 1.28.2
         'oooio-..     `'i:io'             WM: Marco (X11)
```
**version**:
AUR compiled version number 
4.1.0.r374.g8f75ac4e6-1 matching the last commit which is 
8f75ac4e6e9a90c636397fd15c82cb70996c4a2b

**Issue**:
VM created with name "debian 12", after creating, go to view details, over view, change name from "debian 12" to "Q4OS FreeXP Debian 12" , click apply, shows error. 

**Log**
```shell
Error applying changes: Guest name 'Q4OS FreeXP Debian 12' can not contain ' ' character.

Traceback (most recent call last):
  File "/usr/share/virt-manager/virtManager/details/details.py", line 1301, in _config_apply
    success = self._apply_overview()
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/virt-manager/virtManager/details/details.py", line 1395, in _apply_overview
    self.vm.rename_domain(self.widget("overview-name").get_text())
  File "/usr/share/virt-manager/virtManager/object/domain.py", line 565, in rename_domain
    Guest.validate_name(self.conn.get_backend(), str(new_name))
  File "/usr/share/virt-manager/virtinst/guest.py", line 88, in validate_name
    XMLBuilder.validate_generic_name(_("Guest"), name)
  File "/usr/share/virt-manager/virtinst/xmlbuilder.py", line 527, in validate_generic_name
    raise ValueError(msg)
ValueError: Guest name 'Q4OS FreeXP Debian 12' can not contain ' ' character.
```

@crobinso We were able to swap the space validation for the `/` character and also updated `tests/test_xmlparse.py` to account for the new validation.
https://github.com/AbhinavTiruvee/virt-manager/blob/allow-space-disallow-slash/virtinst/xmlbuilder.py#L514
https://github.com/AbhinavTiruvee/virt-manager/blob/allow-space-disallow-slash/tests/test_xmlparse.py#L1020

However, we are getting some errors when trying to run the test suite to confirm that the changes work. Any help with running the test suite would be greatly appreciated. Once we pass the test suite, I'll be sure to open a PR.
`ConftestImportFailure: ValueError: Namespace Libosinfo not available (from /home/virt-manager/tests/conftest.py` and
`ImportError while loading conftest '/home/virt-manager/tests/conftest.py'.
tests/__init__.py:19: in <module>
    from virtinst import buildconfig
virtinst/__init__.py:10: in <module>
    gi.require_version("Libosinfo", "1.0")
/usr/lib/python3/dist-packages/gi/__init__.py:122: in require_version
    raise ValueError('Namespace %s not available' % namespace)
E   ValueError: Namespace Libosinfo not available`

--- a/tests/test_xmlparse.py
+++ b/tests/test_xmlparse.py
@@ -1021,7 +1021,9 @@ def testXMLBuilderCoverage():
         virtinst.DeviceDisk.validate_generic_name("objtype", None)
 
     with pytest.raises(ValueError):
-        virtinst.DeviceDisk.validate_generic_name("objtype", "foo bar")
+        virtinst.DeviceDisk.validate_generic_name("objtype", "foo/bar")
+
+    assert virtinst.DeviceDisk.validate_generic_name("objtype", "foo bar") is None
 
     # Test property __repr__ for code coverage
     assert "DeviceAddress" in str(virtinst.DeviceDisk.address)

--- a/virtinst/xmlbuilder.py
+++ b/virtinst/xmlbuilder.py
@@ -510,9 +510,8 @@ class XMLBuilder(object):
 
     @staticmethod
     def validate_generic_name(name_label, val):
-        # Rather than try and match libvirt's regex, just forbid things we
-        # know don't work
-        forbid = [" "]
+        # Only character that shouldn't work is '/', matching QEMU
+        forbid = ["/"]
         if not val:
             # translators: value is a generic object type name
             raise ValueError(_("A name must be specified for the %s") % name_label)

